src/XMakeCommandLine/UnitTests/project.json

```diff
@@ -7,28 +7,25 @@
   },
   "frameworks": {
     "net451": {
-      "dependencies": {
-        "xunit.runner.visualstudio": "2.1.0"
-      }
+      "dependencies": {}
     },
     "net46": {
-      "dependencies": {
-        "xunit.runner.visualstudio": "2.1.0"
```

Comment:
The `xunit.runner.visualstudio` package is only used for the VS Test Explorer integration.  You can still run tests during the build without it, either using the console runner (which is pretty much what we do for .NET Core), or the [xunit.runner.msbuild](http://xunit.github.io/docs/running-tests-in-msbuild.html) package, which has an MSBuild task you can use.